simlify auto-install instructions (#867)
withastro/docs: src/pages/en/install/auto.md

pnpm create astro@latest
```

The `create-astro` wizard will walk you through setting up a new Astro project, including:

Depending on your package manager, you may be prompted to confirm you want to install `create-astro@latest`. You will then be asked to specify a project folder (e.g. `./my-astro-site`) which will create a new directory.

### Choose a Starter Template
You will then see a short list of starter templates to choose from:
- `Just the basics`: A great starter template for anyone wanting to explore Astro.
- `Blog`, `Documentation`, `Portfolio`: opinionated themes for specific use-cases.
- `Completely empty`: A template that just includes the bare minimium to get started.

Use the arrow keys (up and down) to navigate to the template you want to install, then press return (enter) to submit.

:::tip[Online previews]
Want to check out the templates in your browser before choosing? Visit [astro.new](https://astro.new/)

> :::

Author: I agree! I'll add that back

> ## 2. Install NPM Dependencies

Reviewer: See my proposed change in Next Steps that I think can avoid the need for this section.

I do like the thought of `astro add` as a "next step" afterthought, once your project is already set up. Less to go wrong just getting the project working to start!  ;)

If you already installed your dependencies during `create-astro` setup, then you can skip this and move on to the next step.

Astro is distributed via the [npm ecosystem](https://www.npmjs.com/package/astro), so you will need to install it using `npm` or another npm package manager of your choice. 

When you are satisfied with your selection, press return (enter) to submit.

:::note
These integrations, and any [Astro community integrations](https://astro.build/integrations/), can also be added later by following the instructions in our [integrations guide](/en/guides/integrations-guide/).
:::

After selecting your integrations to add, you should see the following terminal message notifying you of the changes that `create-astro` will apply to your project's `astro.config.mjs`:

```bash
Astro will make the following changes to your config file:
```

This message should show that your chosen integrations have been successfully added to your project configuration. (If not, you can always add them manually later.)

### Initialize a `.git` Repository (optional)

> At this final step, you can choose to initialize a git repository in your new directory. This is optional, but useful if you plan to use the tool [Git](https://git-scm.com/) for your project.

Author: I'm really glad you brought this up since this is the main thing that I'm excited about with this PR! I agree on the primary goal of making sure that we are catering to beginners, including someone who doesn't know what Git is. 

But, I'm introducing a second goal here in this PR: The thing that we're documenting (in this case, `create-astro`) should also be designed with this same user in mind. If that user doesn't know what Git is, then the prompt will confuse them, and they'd have to know to stop what they're doing, know to leave the terminal and switch to their web browser, know to read this section, and then they'd feel better. That's better than nothing but also not ideal.

Instead, I spent the time in https://github.com/withastro/astro/pull/3715 to make sure the tool itself was super beginner friendly. You can watch the video to see it in action, but now if you say no to the prompt it says _"Sounds good! You can come back and run **git init** later."_ I also just updated the question to include the useful bit you added here, about why someone may want to say "yes" to the prompt.

Does that make sense? Docs shouldn't be used to paper over confusing bits of the thing that its documenting, when we can focus on improving the thing to be less confusing. Not only is it a better flow for the user, but it also covers the user who gives up instead of flipping back to the docs.

Watching you all have to document how the multiselect worked was another really good example of this!

> At this final step, you can choose to initialize a git repository in your new directory. This is optional, but useful if you plan to use the tool [Git](https://git-scm.com/) for your project.

Reviewer 2: Perfect! Yes this makes sense. Really happy to see this feed back into the CLI design. Thank you 💜

pnpm create astro@latest
```

The `create-astro` wizard will walk you through setting up a new Astro project, by asking the following questions:

1. Where would you like to create your new Astro project?
    
    Astro must be installed in an empty folder. Type in a name for your new project directory.
    
    ```bash

>     ./my-astro-site

Reviewer: ```suggestion
    my-astro-site
```
punctiation not required when entering a folder name

    ```

> 1. Which template would you like to use? (browse all templates on [astro.new](https://astro.new/)).

Reviewer: ```suggestion
1. Which template would you like to use?
```
We can remove this now that we have links/descriptions below, and we have the tip at the top of the page.

    
    Select from the following list:
    - [Just the Basics](https://github.com/withastro/astro/tree/main/examples/basics) - includes a typical project structure as a model
    - [Blog](https://github.com/withastro/astro/tree/main/examples/blog) - includes an index page and a folder with first blog post

>     - [Docs](https://github.com/withastro/astro/tree/main/examples/docs) - includes sidebar navigation, "On this page" contents and more!

Reviewer: ```suggestion
    - [Docs](https://github.com/withastro/astro/tree/main/examples/docs) - includes sidebar nav, page table of contents and multi-language support
```

>     - [Portfolio](https://github.com/withastro/astro/tree/main/examples/portfolio) - styled and ready to go portfolio and about pages!

Reviewer: ```suggestion
    - [Portfolio](https://github.com/withastro/astro/tree/main/examples/portfolio) - styled and ready to go with portfolio and about pages
```

    - [Completely Empty](https://github.com/withastro/astro/tree/main/examples/minimal) - just a single page

> 1. Would you like to run npm install? (Y/n)

Reviewer: ```suggestion
1. Would you like to run npm (yarn, pnpm) install? (Y/n)
```
This message will differ based on the package manager

Reviewer 2: Would you mind if this deviated slightly from the CLI output?

Could say

```suggestion
1. Would you like to install dependencies? (Y/n)
```

Reviewer 2: Or even
```suggestion
1. Would you like to install dependencies with your package manager? (Y/n)
```

Reviewer: Yup, that'll do me!  💜

# pnpm
pnpm install
```

> ## 3. Start Astro ✨

Author: I'll be sure to update these if we reorganize!

If you aren't able to open your project in the browser, go back to the terminal where you ran the `start` command to see what went wrong.

## 4. Deploy to the web

It's time to deploy your project to the web! Run the `build` command in your project to build your static website to a new `dist/` folder in your project.

```bash
# npm
npm run build

# yarn
yarn build

# pnpm
pnpm run build
```

When the command finishes, you should have a new `dist/` folder in your project that you can deploy directly to your favorite web host.

To get started hosting your website for free, check out our proud hosting partner, [Netlify](https://www.netlify.com/). For instructions on deploying to whatever host you choose, read our detailed [deployment guide](/en/guides/deploy/).

> 

Author: See comment above, which is kind of the same conversation about "where do we draw the line for the installation tutorial?". I'll resolve this thread so that we keep the convo to one thread, but happy to revisit this based on that outcome.

Astro will listen for live file changes in your `src/` directory, so you will not need to restart the server as you make changes during development.

If you aren't able to open your project in the browser, go back to the terminal where you ran the `start` command and look to see if an error occurred, or if your project is being served at a different URL than the one linked to above.

## Next Steps

Success! You are now ready to start building with Astro! 🥳

Here are a few topics that we recommend exploring next. You can read them in any order. You can even leave our documentation for a bit and go play in your new Astro project codebase, coming back here whenever you run into trouble or have a question.

> 📚 **Add a framework:** Learn how to extend Astro with support for React, Svelte, Tailwind and more in our [Integrations guide](/en/guides/integrations-guide/).

Reviewer: ```suggestion
📚 **Add a framework:** Learn how to extend Astro with support for React, Svelte, Tailwind and more with `npx astro add` in our [Integrations guide](/en/guides/integrations-guide/).
```
@delucis - Do you think this strikes a good balance? It doesn't exactly SAY run the command... it says "Go learn about the command!" But those who are adventurous/confident can just run the command right away,

Reviewer 2: Don't love the double "with". Maybe 
```suggestion
📚 **Add a framework:** Learn how to extend Astro with support for React, Svelte, Tailwind and more using `npx astro add` in our [Integrations guide](/en/guides/integrations-guide/).
```

> 📚 **Build your site:** Learn how to build and deploy an Astro project to the web in our [Deployment guide](/en/guides/deploy/).

Reviewer: ```suggestion
📚 **Deploy your site:** Learn how to build and deploy an Astro project to the web in our [Deployment guide](/en/guides/deploy/).
```
We've seen very recently a stream with someone wanting to deploy *instantly*, even before editing the site. And, our full tutorial takes this approach - get it created and deploy to Netlify so you *have something* tangible immediately.

Reviewer 2: +1 as we're linking to the deploy guide _and_ still mention building.

Reviewer: My other thought was to change the link to deploy directly to the "build your site" anchor link, but I'm leaning towards emphasizing deploy here, with building being mentioned, and available at the link. (And, not sending someone to the bottom of a page.)

Reviewer 2: 👍
